less/switchControls.less

```diff
@@ -36,6 +36,19 @@
       }
     }
   }
+  
+  &.small {
```

Comment:
In my search, no other component currently uses the _small_ variant apart from `about:preferences`, this moves the previous style override to the shared component as the _large_ variant already was.